### PR TITLE
[TypeScript SDK] Fix type definition for AuthorityQuorumSignInfo

### PR DESCRIPTION
**Pull request by 666lcz**
The definition of [AuthorityQuorumSignInfo](https://github.com/MystenLabs/sui/blob/e9d494e4ca65f877581817b9dd9d5d942fedba7e/crates/sui-types/src/crypto.rs#L512) was updated in https://github.com/MystenLabs/sui/commit/e9d494e4ca65f877581817b9dd9d5d942fedba7e#diff-ccd99b30fee16b246ae935ee7b8c4d336530cc6962dcd611695d73f9493e8876 but the TypeScript was not updated, this PR fixes it.

## Testing
CleanShot 2022-07-07 at 23 02 43

### CODE REVIEW

File: crates/sui-open-rpc/samples/objects.json [hunk 1]
```diff
@@ -9,22 +9,22 @@
         "fields": {
           "description": "An NFT created by the Sui Command Line Tool",
           "id": {
-            "id": "0x0bd2ba409b00487652fe84aa1ffe61cb1623062f",
+            "id": "0xc3e9d83dcae448e0dd11e4b6f335ff5c799e8634",
```
Comment (gegaowp) [inline crates/sui-open-rpc/samples/objects.json:12]: curious why do we want to change these IDs and addresses?

Comment (gegaowp) [reply crates/sui-open-rpc/samples/objects.json:12]: are they auto generated?

**Review by gegaowp — APPROVED** [review]
thanks for the quick fix, wanted to put up one though :)

File: crates/sui-open-rpc/samples/objects.json (hunk 1 above)
Comment (666lcz) [reply crates/sui-open-rpc/samples/objects.json:12]: Yes, they are generated through https://github.com/MystenLabs/fastnft/blob/e7778f4af425a909c4f453eb2c5104ccead64f54/crates/generate-json-rpc-spec/src/main.rs#L79

Comment (gegaowp) [reply crates/sui-open-rpc/samples/objects.json:12]: @666lcz sry what I meant was, if I want to generate this, how do I do that? thanks!

Comment (666lcz) [reply crates/sui-open-rpc/samples/objects.json:12]: You can run `cargo -q run --example generate-json-rpc-spec -- record` after https://github.com/MystenLabs/sui/pull/3103 is merged. You can also just hit the run button in VSCode when you open the file
CleanShot 2022-07-08 at 10 21 20

**Pull request merged**